cmds/cbfs/cbfs.go

```diff
@@ -1,37 +1,218 @@
 package main
 
 import (
+	"bytes"
+	"encoding/binary"
 	"fmt"
 	"log"
+	"os"
 
 	"github.com/linuxboot/cbfs/pkg/cbfs"
 	flag "github.com/spf13/pflag"
 )
 
-var debug = flag.BoolP("debug", "d", false, "enable debug prints")
+//CommandlineArgs holds the parsed command line arguments
+type CommandlineArgs struct {
+	Debug           bool
+	Image           string
+	Command         string
+	CustomArguments map[string]*CustomCommandlineArg
+}
 
-func main() {
-	flag.Parse()
+type CustomCommandlineArg struct {
```

Comment:
run go vet and golint and do what it says :-) it will complain about this.

Reply:
removed from this PR, feature will be added in the next PR as well as tests.